app/controllers/users_controller.rb

```diff
@@ -162,7 +173,10 @@ def onboarding_checkbox_update
 
     current_user.saw_onboarding = true
     authorize User
-    render_update_response
+    if current_user.save
+      success = true
```

Comment:
I don't think we gain much from this variable, I'd inline this:

```ruby
     render_update_response(current_user.save)
```

Reply:
Definitely!